modules/aspect_bazel_lib/1.21.4/MODULE.bazel

```diff
@@ -0,0 +1,23 @@
+"aspect-build/bazel-lib"
+
+module(
+    name = "aspect_bazel_lib",
+    compatibility_level = 1,
+    version = "1.21.4",
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.3.0")
+bazel_dep(name = "platforms", version = "0.0.4")
+bazel_dep(name = "stardoc", repo_name = "io_bazel_stardoc", version = "0.5.0")
```

Comment:
May have missed this before, but does this really have to be a non-dev dependency?

Reply:
Yes, we have a library helping users document their rules, https://docs.aspect.build/rules/aspect_bazel_lib/docs/